sopt/src/main/scala/dagr/sopt/util/ParsingUtil.scala

```diff
@@ -0,0 +1,141 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2015-2016 Fulcrum Genomics LLC
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package dagr.sopt.util
+
+import java.lang.reflect.Modifier
+
+import dagr.commons.reflect.ReflectionUtil
+import dagr.commons.util.ClassFinder
+import dagr.sopt.cmdline._
+import dagr.sopt.clp
+
+import scala.collection.JavaConversions._
+import scala.collection.immutable.Map
+import scala.reflect._
+import scala.reflect.runtime.universe._
+
+/** Variables and Methods to support command line parsing */
+object ParsingUtil extends Configuration  {
```

Comment:
Again, not new, but I've never liked that scala makes you say extend for the first trait if you're not extending another class.  How do you feel about doing this instead?:
```scala
object ParsingUtil extends Object with Configuration
```

I think that makes it clearer that you're "adding" configuration, rather than being an extension of it.